src/System.Runtime.Caching/src/Configurations.props

```diff
@@ -1,14 +1,8 @@
 ﻿<Project>
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
-      netcoreapp2.0-Windows_NT;
-      netcoreapp2.0-Unix;
```

Comment:
Do you need to tell it to not harvest these for newer packages?  (I seem to recall that if the package carries a netcoreapp2.0 asset then when deciding what to use on netcoreapp5.0 it'll pick "older version of me" over "this netstandard thing that I'm compatible with")

Reply:
Harvesting of files is never automatic and this package doesn't explictly harvest anything : https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Caching/pkg/System.Runtime.Caching.pkgproj.  The only thing we do automatically is examine what the package supported and which versions it exposed.